.ci_config

```diff
@@ -0,0 +1 @@
+Subproject commit 31f2b420a0816e459273078445dacd805d9501e3
```

Comment:
An example of the new travis ci usage can be found here: https://github.com/ros-industrial/industrial_core/blob/indigo-devel/.travis.yml